composer.json

```diff
@@ -82,7 +82,7 @@
                 "type:bower-asset"
             ],
             "web/modules/contrib/{$name}": ["type:drupal-module"],
-            "web/profiles/{$name}": ["type:drupal-profile"],
+            "web/profiles/contrib/{$name}": ["type:drupal-profile"],
```

Comment:
Existing sites all skip the "contrib" subdirectory. I assume converting them will be mostly painless.

Reply:
They'll all need a cache rebuild to detect the new path, but I don't expect other problems.